gensim/models/wrappers/ldamallet.py

```diff
@@ -245,11 +247,22 @@ def show_topic(self, topicid, topn=10):
     def print_topic(self, topicid, topn=10):
         return ' + '.join(['%.3f*%s' % v for v in self.show_topic(topicid, topn)])
 
+    #########  function to check the version of mallet can be generalised to all the problematic version  ######
+    def check_version(self, direc_path):
+        xml_path = direc_path.split("bin")[0]
+        soup=BeautifulSoup(open(xml_path+"pom.xml").read())
+        if soup.find("version").text == "2.0.7-SNAPSHOT":
+            return True
```

Comment:
Please return version number. There is no problematic version. Both versions are valid.